### PR TITLE
Added version checksums (7.1.10, 7.2.0, 7.2.1, 7.2.2, 7.2.3)

### PR DESCRIPTION
**Pull request by katbyte**
Added version checksums for the latest versions and changed the url generation logic to match the new software url format.

### CODE REVIEW

File: libraries/jira.rb [hunk 1]
```diff
@@ -67,26 +67,27 @@ def jira_artifact_url
...
           product = "#{base_url}/atlassian-jira-#{node['jira']['flavor']}-#{version}-jira-#{version}"
-        when 'core'
+        else
           product = "#{base_url}/atlassian-jira-#{node['jira']['flavor']}-#{version}"
         end
       end
 
       # Return actual URL
       case node['jira']['install_type']
-      when 'installer'
```
Comment (legal90) [inline libraries/jira.rb]: Indent `when` as deep as `case`
https://github.com/bbatsov/ruby-style-guide#indent-when-to-case

Comment (katbyte) [inline libraries/jira.rb]: Corrected

**Comment by jsibo** [comment]
Bump - can we fix the checks & merge this asap? @katbyte @mvdkleijn

**Comment by katbyte** [comment]
I don't believe the checks are failing do to anything i have committed, seems commit 9a16564 broke the build back in march and it has been broken ever since.

I don't use travis CI so I'm not sure what the problem is or how i could fix it.

**Pull request merged**